### ISSUE TITLE
Question-source code of @types/showdown:1.4.31

### ISSUE THREAD

**Issue by iamfariba** [DefinitelyTyped/DefinitelyTyped#37990]
please let me know where is the link to download source code of @types/showdown version 1.4.31?

https://www.npmjs.com/package/@types/showdown/v/1.4.31 for the repository has this link https://github.com/DefinitelyTyped/DefinitelyTyped but this contains only master version,

Please let me know source code address for @types/showdown version 1.4.31.

Best Regards

**Comment by mastermatt** [comment]
Here is a link to the source that published as 1.4.31 https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f4f57de0b5/showdown/showdown.d.ts

According to [npm](https://www.npmjs.com/package/@types/showdown/v/1.4.31), that version was published on 2019-09-19, so I used `git log` to find a sha from just after that.

Note that the header in that file has 1.4.1 as the DT types don't match patch versions with the lib unless a contributor manually updates it. 

If you just wanted to view the types, you could have installed the lib locally then inspected the file.

```
npm i @types/showdown@1.4.31
vim ./node_modules/@types/showdown/index.d.ts
```
Or used a service like [unpkg](https://unpkg.com/@types/showdown@1.4.31/index.d.ts).

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
